extensions/src/main/java/org/mapstruct/extensions/spring/converter/ConverterMapperProcessor.java

```diff
@@ -155,7 +166,7 @@ private void writeAdapterClassFile(
       final Set<? extends TypeElement> annotations, final RoundEnvironment roundEnv) {
     final MutablePair<String, String> packageAndClass =
         MutablePair.of(
-            ConverterMapperProcessor.class.getPackage().getName(), "ConversionServiceAdapter");
+            "org.mapstruct.extensions.spring", "ConversionServiceAdapter");
```

Comment:
This shouldn't be a fixed package name. I understand why one would that for the `@AutoMap` feature, but we can't have that as a general setting.